SemanticSearch2.md

```diff
@@ -176,7 +176,9 @@ The document response is divided into two parts. First is the parsed_input node
 		- name (canonical string)
```

Comment:
The following format may be clearer.
- filed name (type): explanation.
So this will be -name (String): canonical form.
Also apply to the below fields.